packages/api/endpoints/dashboard.js

```diff
@@ -14,7 +14,7 @@ const { s3 } = require('@cumulus/aws-client/services');
  * @param {Object} res - express response object
  * @returns {Promise<Object>} the promise of express response object
  */
-async function handleGetRequst(req, res) {
+async function handleGetRequest(req, res) {
```

Comment:
Is this called anywhere else (e.g. a test?) that might also need updating?

Reply:
I verified the refactor in the Core codebase (text search for references)  - I don't believe it's used in the dashboard directly.